lib/internal/streams/readable.js

```diff
@@ -230,6 +243,12 @@ Readable.prototype[EE.captureRejectionSymbol] = function(err) {
 // similar to how Writable.write() returns true if you should
 // write() some more.
 Readable.prototype.push = function(chunk, encoding) {
+  if (this[kConsume] && chunk !== null && !this[kReading]) {
+    // Fast path.
+    encoding = encoding || this._readableState.defaultEncoding;
```

Comment:
```suggestion
    encoding ||= this._readableState.defaultEncoding;
```
Or is it to be able to backport this to 14.x?

Reply:
We can backport parts of it to 14.